server/portal/apps/workspace/api/views.py

```diff
@@ -230,8 +230,8 @@ def get(self, request, *args, **kwargs):
             user_job_ids = all_user_job_ids[offset:offset + limit]
             if user_job_ids:
                 data = agave.jobs.list(query={'id.in': ','.join(user_job_ids)})
-                # re-order agave job info to match our time-ordered jobs
-                data = [next(job for job in data if job["id"] == id) for id in user_job_ids]
+                # filter agave jobs response to match our time-ordered jobs
+                data = [job for job in data if job["id"] in user_job_ids]
```

Comment:
I think we still gotta reorder `data` based on the `order` of `user_job_is` and this change might tweak that.

I think that even though we give tapis an ordered list of job-ids to return, tapis returns the jobs in a different order.  I'll run and check out tomorrow locally to see if that is still the case.

Reply:
In addition to the above, there is also an issue that would occur as we assume that if there are N jobs requested and if <N jobs are returned then there is no need to request more during scrolling.

I think we can just ignore this issue as it's such an edge case (and would be problematic to solve) but curious on your opinion.